app/src/Services/modules/events/fetchAll.ts

```diff
@@ -20,6 +20,6 @@ export default (
 ) =>
   build.query<Events, string>({
     query: () =>
-      `${Config.STA_BASE_URL}${Config.STA_API_VERSION}/events/schedule/${range}`,
+      `${Config.STA_BASE_URL}/${Config.STA_API_VERSION}/events/schedule/${range}`,
```

Comment:
I don't think we should include this and the other similar change below, as it should work without this change using e.g. `STA_BASE_URL` as `'https://the-sta.com'` and `STA_API_VERSION` as `'/v1'` -- the existing string literal should make this e.g. 'https://the-sta.com/v1/events/schedule/past'

Are you still having an issue locally?  If so could we debug that separately?  Happy to help figure out what's going wrong

Reply:
I didn't notice this had slipped through from the other PR. Thought I'd stashed it. Removed from the projects folder too.